src/components/BasicAccordion/BasicAccordion.model.ts

```diff
@@ -0,0 +1,6 @@
+export interface BasicAccordionProps {
+  title: string;
+  description: string;
```

Comment:
Accordion child/detail can include the React element too, please modify the props to accept children either as text/React.Node